src/components/standalone/dashboard/SystemInfoCard.vue

```diff
@@ -142,10 +142,6 @@ function goToUpdates() {
 async function getUpdatesStatus() {
   try {
     systemUpdateData.value = (await ubusCall('ns.update', 'check-system-update')).data
-  } catch (err: any) {
```

Comment:
I'd add a comment like `// Ignore error: toast error notification is enough here` for future reference